src/zone.ts

```diff
@@ -368,7 +396,7 @@ class Zone extends ServiceObject<Zone> {
     const config = typeof configOrCallback === 'object' ? configOrCallback : {};
     callback =
       typeof configOrCallback === 'function' ? configOrCallback : callback;
-    super.get(config, callback!);
+    ServiceObject.prototype.get.call(this, config, callback!);
```

Comment:
Same as above, wondering why we can't just use `super`

Reply:
likewise, I'm wondering could we perhaps make `ZoneServiceObject` inherit from `ServiceObject`, and address both the `instanceof` issue and Justin's concerns about `super`?